flotta-agent.spec

```diff
@@ -45,8 +45,12 @@ getent passwd %{flotta_user} >/dev/null || useradd -g %{flotta_user} -s /sbin/no
 %post
 systemctl enable --now nftables.service
 loginctl enable-linger %{flotta_user}
-systemctl start user@$(getent passwd %{flotta_user} | cut -d: -f3).service
+systemctl start user@$(id -u %{flotta_user}).service
+
+# HACK till https://bugzilla.redhat.com/show_bug.cgi?id=2060702 is fixed
+setenforce 0
```

Comment:
wait, here shouldn't we use enable instead of start?

Reply:
@machacekondra can you refer to start vs enable?
do we have script for user flotta service?

Reply:
Oh, right we should do both:
```
systemctl --machine %{flotta_user}@.host --user enable podman.socket
systemctl --machine %{flotta_user}@.host --user start podman.socket
```

Reply:
IIUC @eloycoto referred to the line above:
```
systemctl start user@$(id -u %{flotta_user}).service
```
vs
```
systemctl enable --now user@$(id -u %{flotta_user}).service
```

Reply:
Yeah, `enable --now` is more elegant.

Reply:
`systemctl start user@$(id -u %{flotta_user}).service` was removed.
it is being started by `loginctl enable`

Reply:
The bug was last updated on `2022-03-10`. It seems like it is not a priority to fix it. Maybe we could contribute a fix for this BZ?